CMakeLists.txt

```diff
@@ -96,11 +96,21 @@ if (OGRE_FOUND)
 endif()
 
 #--------------------------------------
-# Find OGRE2
+# Find OGRE2: first try to find OGRE2 built with PlanarReflections support and
+# fallback to look for OGRE2 without it. Both seems to works for ign-rendering.
+# See https://github.com/ignitionrobotics/ign-rendering/issues/597
 ign_find_package(IgnOGRE2 VERSION 2.2.0
+    COMPONENTS HlmsPbs HlmsUnlit Overlay PlanarReflections
+    PRIVATE_FOR ogre2
+    QUIET)
+
+if (NOT OGRE2_FOUND)
```

Comment:
maybe check to see if `${OGRE2-PlanarReflections}` equals `"OGRE2-PlanarReflections-NOTFOUND"` here?

Reply:
uh much better, thanks Ian [3fd7e1d](https://github.com/ignitionrobotics/ign-rendering/pull/605/commits/3fd7e1d06f79454000e9ca0021c5b9f54750abe7).